test/data/hcph_multishell.txt

```diff
@@ -0,0 +1,4 @@
00 1000.000000 3000.000000 3000.000000 3000.000000 3000.000000 3000.000000 3000.000000 2000.000000 3000.000000 2000.000000 2000.000000 2000.000000 3000.000000 2000.000000 3000.000000 2000.000000 1000.000000 3000.000000 2000.000000 0.000000 3000.000000 1000.000000 3000.000000 3000.000000 2000.000000 2000.000000 700.000000 3000.000000 3000.000000 2000.000000 3000.000000 700.000000 3000.000000 2000.000000 2000.000000 3000.000000 2000.000000 2000.000000 3000.000000 2000.000000 3000.000000 3000.000000 2000.000000 3000.000000 2000.000000 3000.000000 1000.000000 1000.000000 3000.000000 1000.000000 2000.000000 700.000000 2000.000000 1000.000000 2000.000000 3000.000000 3000.000000 700.000000 3000.000000 2000.000000 3000.000000 1000.000000 3000.000000 2000.000000 1000.000000 2000.000000 0.000000 3000.000000 1000.000000 1000.000000 3000.000000 3000.000000 3000.000000 1000.000000 3000.000000 2000.000000 2000.000000 2000.000000 3000.000000 3000.000000 2000.000000 2000.000000 3000.000000 700.000000 3000.000000 3000.000000 2000.000000 1000.000000 2000.000000 3000.000000 3000.000000 3000.000000 2000.000000 3000.000000 3000.000000 3000.000000 1000.000000 3000.000000 2000.000000 1000.000000 2000.000000 2000.000000 2000.000000 3000.000000 3000.000000 3000.000000 1000.000000 3000.000000 3000.000000 3000.000000 3000.000000 3000.000000 0.000000 3000.000000 3000.000000 3000.000000 2000.000000 2000.000000 700.000000 3000.000000 2000.000000 1000.000000 1000.000000 2000.000000 3000.000000 3000.000000 1000.000000 1000.000000 3000.000000 3000.000000 3000.000000 3000.000000 2000.000000 3000.000000 2000.000000 2000.000000 3000.000000 2000.000000 2000.000000 2000.000000 1000.000000 3000.000000 3000.000000 3000.000000 3000.000000 3000.000000 3000.000000 2000.000000 3000.000000 2000.000000 2000.000000 2000.000000 3000.000000 2000.000000 3000.000000 2000.000000 1000.000000 3000.000000 2000.000000 0.000000 3000.000000 1000.000000 3000.000000 3000.000000 2000.000000 2000.000000 700.000000 3000.000000 3000.000000 2000.000000 3000.000000 700.000000 3000.000000 2000.000000 2000.000000 3000.000000 2000.000000 2000.000000 3000.000000 2000.000000 3000.000000 3000.000000 2000.000000 3000.000000 2000.000000 3000.000000 1000.000000 1000.000000 3000.000000 1000.000000 2000.000000 700.000000 2000.000000 1000.000000 2000.000000 3000.000000 3000.000000 700.000000 3000.000000 2000.000000 3000.000000 1000.000000 3000.000000 2000.000000 1000.000000 2000.000000 0.000000 3000.000000 1000.000000 1000.000000 3000.000000 3000.000000 3000.000000 1000.000000 3000.000000 2000.000000 2000.000000 2000.000000 3000.000000 3000.000000 2000.000000 2000.000000 3000.000000 700.000000 3000.000000 3000.000000 2000.000000 1000.000000 2000.000000 3000.000000 3000.000000 3000.000000 2000.000000 3000.000000 3000.000000 3000.000000 1000.000000 3000.000000 2000.000000 1000.000000 2000.000000 2000.000000 2000.000000 3000.000000 3000.000000 3000.000000 1000.000000 3000.000000 3000.000000 3000.000000 3000.000000 3000.000000
```

Comment:
Do we need this file? If we're not testing other than the CLI implementation, an empty file or a file with random content should be good enough, right?